docs/ml-guide.md

```diff
@@ -951,4 +951,71 @@ model.transform(test)
 {% endhighlight %}
 </div>
 
-</div>
\ No newline at end of file
+</div>
+
+## Example: Saving and Loading a Previously Created Model Pipeline
+
+Often times it is worth it to save a model to disk for usage later. In Spark 1.6, similar model import/export functionality was added to the Pipeline API. Most basic transformers are supported as well as some of the more basic ML Models such as:
```

Comment:
"for later use" instead of "for usage later" sounds better to me.
"In Spark 1.6, **a** model import/export" instead of "similar"
"ML **m**odels"